project-management/src/main/java/life/qbic/projectmanagement/application/authorization/QbicOidcUser.java

```diff
@@ -73,10 +69,19 @@ public String getName() {
   }
 
   public String getOidcId() {
-    return originalAuthName;
+    return Optional.ofNullable(originalAuthName).orElse(qbicUserInfo.oidc());
```

Comment:
I am not sure you want to do this... This might interfere with DefaultOidcUser. It is assumed that when the login did not work we can safely load the information from our database. This is not correct. Am I missing something?